concourse/steps/replicate_secrets.mako

```diff
@@ -99,11 +103,32 @@ replicate_secrets(
 
 logger.info('generating cfg element status report')
 
-status_reports = cmu.generate_cfg_element_status_reports('${cfg_repo_relpath}')
+status_reports = cmu.generate_cfg_element_status_reports(
+  cfg_dir='${cfg_repo_relpath}',
+  element_storage='${cfg_repo_url}',
+)
 cmr.create_report(status_reports)
 cfg_report_summary_gen = cmr.cfg_element_statuses_storage_summaries(status_reports)
 cfg_responsible_summary_gen = cmr.cfg_element_statuses_responsible_summaries(status_reports)
 
+% if compliance_reporting_repo_url:
+## use default from `CONCOURSE_CURRENT_CFG` once it refers to proper cfg_set
+delivery_svc_client = ccc.delivery.client(
+  cfg_name='${delivery_endpoints_cfg_name}',
+  cfg_factory=cfg_factory,
+)
+
+report_cfg_policy_status(
+  cfg_factory=cfg_factory,
+  status_reports=status_reports,
+  compliance_reporting_repo_url='${compliance_reporting_repo_url}',
+  delivery_svc_client=delivery_svc_client,
+  github_issue_template_cfgs=${github_issue_template_cfgs},
+)
+% else:
+logger.info('will not generate reporting issues')
```

Comment:
ideally also state _why_ (absence of reporting-repo-url) to make the info-message more helpful (please, in general, consider this for logging output; similar to comments in source-code, please always consider expressing WHY over WHAT + non-obvious context-information (think of log-readers that do not have a mapping to the logging source code))